assets/src/edit-story/elements/video/display.js

```diff
@@ -78,6 +78,8 @@ function VideoDisplay({ previewMode, box: { width, height }, element }) {
 
   videoProps.crossOrigin = 'anonymous';
 
+  const muted = Boolean(resource?.isMuted);
```

Comment:
Not sure if we need this. Made sense to me.

Reply:
We'd want to update `video/output.js` as well to set the `nooaudio` attribute on the `amp-video` -- **not** `muted`, which is deprecated in `amp-video`

Reply:
> not muted, which is deprecated in amp-video

You know I looked adding `muted` to `amp-video` and didn't because I noticed that. I will add it.